lib/base/application.cpp

```diff
@@ -746,11 +754,14 @@ void Application::SigAbrtHandler(int)
 		Log(LogCritical, "Application")
 			<< "Icinga 2 has terminated unexpectedly. Additional information can be found in '" << fname << "'" << "\n";
 
+		ofs << "Caught SIGABRT." << "\n"
+			<< "Current time: " << Utility::FormatDateTime("%Y-%m-%d %H:%M:%S %z", Utility::GetTime()) << "\n"
+			<< "\n";
+
 		DisplayInfoMessage(ofs);
 
-		StackTrace trace;
-		ofs << "Stacktrace:" << "\n";
-		trace.Print(ofs, 1);
+		ofs << "\n" << "Stacktrace:" << "\n"
+			<< boost::stacktrace::stacktrace() << "\n";
```

Comment:
I'd not

1. separate " " and " " with << if they're on the same line
2. put stuff on different lines in the code just because it prints different lines of text

Reply:
So you'd just do `ofs << "\nStacktrace:\n" << boost::stacktrace::stacktrace() << "\n";` in a single line? The existing code seems to prefer putting "\n" in separate strings, but I also have merged then into the neighboring strings in other places, so this PR is not totally consistent in this way so far.

Reply:
Yes, _I'd_ do so.

Reply:
Strings should be more compact in the most recent version I pushed.

Reply:
ofs << "Caught SIGABRT.**" << "**\n" still looks strange to me.

Reply:
Pushed an additional commit that changes this there and in other places as this was direct copy-paste from elsewhere in that file: https://github.com/Icinga/icinga2/blob/ff5385bbb5f0ab3f32b52ec4cdf94b4c1a452972/lib/base/application.cpp#L856-L858